### PR TITLE
el-loading,fix 嵌套使用fullscreen属性时,前400ms 会出现背景色重叠的问题

### PR DESCRIPTION
**Pull request by BigPengZai**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by msidolphin** [comment]
Can you provide a minimized reproduction demo?

**Comment by BigPengZai** [comment]
> Can you provide a minimized reproduction demo?

```
<template>
  <div
    v-loading.fullscreen.lock="fullscreenLoading"
    class="container"
    element-loading-background="rgba(0, 0, 0, 0.3)"
  >
    <el-button
      v-loading.fullscreen.lock="fullscreenLoading"
      element-loading-background="rgba(0, 0, 0, 0.3)"
      type="primary"
      @click="openFullScreen1"
    >
      show overlap
    </el-button>
  </div>
</template>

<script lang="ts">
import { defineComponent, ref } from 'vue'

export default defineComponent({
  setup() {
    const fullscreenLoading = ref(false)
    const openFullScreen1 = () => {
      fullscreenLoading.value = true
      setTimeout(() => {
        fullscreenLoading.value = false
      }, 1500)
    }

    return {
      fullscreenLoading,
      openFullScreen1,
    }
  },
})
</script>
<style scoped>
.container {
  display: flex;
  justify-content: center;
  align-items: center;
}
</style>

```

File: packages/components/loading/src/loading.ts [hunk 1]
```diff
@@ -43,15 +43,18 @@ export function createLoadingComponent(options: LoadingOptionsResolved) {
       }
       removeClass(target, 'el-loading-parent--hidden')
     }
+    removeElParent()
+  }
+  function removeElParent(): void {
```
Comment (msidolphin) [inline packages/components/loading/src/loading.ts:48]: The function name should be removeChild

File: packages/components/loading/src/loading.ts [hunk 2]
```diff
@@ -43,15 +43,18 @@ export function createLoadingComponent(options: LoadingOptionsResolved) {
...
     vm.$el?.parentNode?.removeChild(vm.$el)
   }
-
   function close() {
     if (options.beforeClose && !options.beforeClose()) return
 
     const target = data.parent
     target.vLoadingAddClassList = undefined
     afterLeaveFlag.value = true
+    removeElParent()
```
Comment (msidolphin) [inline packages/components/loading/src/loading.ts:57]: I don't think we should remove the child here, we should try remove in service.ts

```ts
if (resolved.fullscreen && fullscreenInstance) {
  fullscreenInstance.remvoeChild()
  fullscreenInstance.close()
}
```

**Comment by msidolphin** [comment]
Please do not use Chinese to write git comments

File: packages/components/loading/src/loading.ts (hunk 2 above)
Comment (BigPengZai) [reply packages/components/loading/src/loading.ts:57]: > 
OK, I accept your proposal. I will modify it immediately. Please accept my changes

**Review by msidolphin — APPROVED** [review]

**Comment by BigPengZai** [comment]
Please check this question

[ElLoading.service  single bug ](https://github.com/element-plus/element-plus/pull/4992)

**Pull request merged**